test/tests/pattern-selection.json

```
@@ -0,0 +1,150 @@
+{
+  "$schema": "https://raw.githubusercontent.com/unicode-org/message-format-wg/main/test/schemas/v0/tests.schema.json",
+  "scenario": "Pattern selection",
+  "description": "Tests for the pattern selection parts of the specification",
```

```suggestion
  "description": "Tests for pattern selection",
```

I think this could be less verbose